src/test/java/showcase/neodymium/tests/slider/SliderTest.java

```diff
@@ -0,0 +1,96 @@
+    public void testHorizontalRightMovement()
+    {
+        // open demo page
+        Selenide.open("https://demos.telerik.com/kendo-ui/slider/index");
+
+        // move the slider to the right
+        // offset for the horizontal movement: 40
+        // offset for the vertical movement: 0
+        // until the value: 8
+        // retries: 5
+        // pause between movements: 3000ms
+        SelenideAddons.dragAndDropUntilCondition($(".balSlider a[role=slider]"), $(".balSlider a[role=slider]"), 40, 0, 3000, 5,
+                                                 Condition.attribute("aria-valuenow", "8"));
+    }
+
+    @Test
+    @Description(value = "Showcase for a horizontal drag and drop of a web slider to the left side.")
+    public void testHorizontalLeftMovement()
+    {
+        // open demo page
+        Selenide.open("https://demos.telerik.com/kendo-ui/slider/index");
+
+        // move the slider to the left
+        // offset for the horizontal movement: -40
+        // offset for the vertical movement: 0
+        // until the value: -8
+        // retries: 5
+        // pause between movements: 3000ms
+        SelenideAddons.dragAndDropUntilCondition($(".balSlider a[role=slider]"), $(".balSlider a[role=slider]"), -40, 0, 3000, 5,
+                                                 Condition.attribute("aria-valuenow", "-8"));
+    }
+
+    @Test
+    @Description(value = "Showcase for a vertical drag and drop of a web slider to the top.")
+    public void testVerticalUpMovement()
+    {
+        // open demo page
+        Selenide.open("https://demos.telerik.com/kendo-ui/slider/index");
+
+        // move the slider to the top
+        // offset for the horizontal movement: 0
+        // offset for the vertical movement: -10
+        // until the value: 16
+        // retries: 5
+        // pause between movements: 3000ms
+        SelenideAddons.dragAndDropUntilCondition($("#equalizer .k-slider-vertical:first-child a"), $("#equalizer .k-slider-vertical:first-child a"), 0, -10,
+                                                 3000, 5, Condition.attribute("aria-valuenow", "16"));
+    }
+
+    @Test
+    @Description(value = "Showcase for a vertical drag and drop of a web slider to the bottom.")
+    public void testVerticalDownMovement()
+    {
+        // open demo page
+        Selenide.open("https://demos.telerik.com/kendo-ui/slider/index");
+
+        // move the slider to the bottom
+        // offset for the horizontal movement: 0
+        // offset for the vertical movement: 10
+        // until the value: -6
+        // retries: 5
+        // pause between movements: 3000ms
+        SelenideAddons.dragAndDropUntilCondition($("#equalizer .k-slider-vertical:first-child a"), $("#equalizer .k-slider-vertical:first-child a"), 0, 10,
+                                                 3000, 5, Condition.attribute("aria-valuenow", "-6"));
+    }
+}
```

Comment:
```suggestion
}

```
Add a new line to the end of the file.

Reply:
Please add a new line.